index.js

```diff
@@ -9,9 +11,35 @@
  *
  */
 
+var lib = require('manifoldjs-lib');
+var libValidationContants = lib.constants.validation;
+
+// Maintain compatibility with original constant definitions
+var validationConstants = {
+  levels:           libValidationContants.levels,
```

Comment:
Yea, that is probably me who leave white space behind, and shouldn't.  I'm dyslexic, so whitespace helps me keep things straight, but I try to remove it before checking in.  Good call.